lib/biola_frontend_toolkit/version.rb

```diff
@@ -1,3 +1,3 @@
 module BiolaFrontendToolkit
-  VERSION = '1.0.0'.freeze
+  VERSION = '1.0.1'.freeze
```

Comment:
Since you're merging this in after the Rails 5 changes you'll have to wait to deploy this until we finish getting the updated WCMS apps out the door.